packages/airnode-examples/integrations/coingecko-http-gateways/config.example.json

```diff
@@ -63,7 +63,9 @@
       "corsOrigins": []
     },
     "oevGateway": {
-      "enabled": false
+      "enabled": true,
+      "maxConcurrency": 20,
+      "corsOrigins": []
```

Comment:
I understand it would be just a copy & paste, most of the examples are that way anyway. But since the examples are at least partially user-facing I think the distinction between the gateways should be made obvious.

Reply:
That's also a way to look at this. The sad part is that the gateway is still part of Airnode and everything that's inside Airnode is user-facing... I am not sure I get why we should distinct between signed and oev gateway and not between http and signed.

Reply:
That's what I wrote in my initial comment. The HTTP and HTTP signed data gateways "emulate" Airnode requests off-chain. The OEV gateway does no such thing, its use case is totally different.

Reply:
I disabled the OEV gateway from the example and created https://github.com/api3dao/airnode/issues/1687